.github/test_db/config/database.yml

```diff
@@ -1,5 +1,5 @@
 test:
- adapter: postgresql
+ adapter: postgis
```

Comment:
this changes are repeated in the other PR.. please, point one PR to the other if they are related